src/main/java/pl/simpleascoding/tutoringplatform/exception/ReviewNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package pl.simpleascoding.tutoringplatform.exception;
+
+public class ReviewNotFoundException extends RuntimeException{
+
+    private static final String ERROR = "Review with id %d not found";
```

Comment:
Please change name of final ERROR to MESSAGE

Reply:
Done